kafka-client/src/main/java/dev/responsive/kafka/internal/stores/CommitBuffer.java

```diff
@@ -75,6 +106,17 @@ class CommitBuffer<K, S extends RemoteTable<K>>
   private final Supplier<Instant> clock;
   private final KeySpec<K> keySpec;
 
+  private final String flushSensorName;
+  private final String flushLatencySensorName;
+  private final String flushFencedSensorName;
+  private final String failedTruncationsSensorName;
+
+  private final MetricName lastFlushMetric;
```

Comment:
we can do this in a follow up, but it'd be nice if we grouped all the metrics together in a single CommitBufferMetrics class just to reduce the verbosity of this (already relatively large) file

Reply:
meant to follow up on this, yeah I would definitely like to consolidate the actual metric/sensor creation into the corresponding metrics group class. Both to reduce the verbosity and to keep all the metrics in one place so it's easy to find them and to tell what metrics are actually being added and how. I left that out of scope for now but I do plan on adding a few more batches of metrics maybe after the window store stuff, at which point I think it makes sense to do this cleanup